verify_docs.py

```diff
@@ -6,22 +8,62 @@
 
 import sys
 import yaml
+import copy
+
+from distutils.version import StrictVersion
+
+def verify_swagger(spec):
+    errored = False
+    for i in spec["paths"]:
+        for j in spec["paths"][i]:
+            k = spec["paths"][i][j]
+            if "summary" not in k:
+                print("Error: summary missing from Swagger doc in: %s " % (i))
+                errored = True
+    return errored
+
+def verify_openapi(spec):
+    errored = False
+    for i in spec["paths"]:
+        for j in spec["paths"][i]:
+            k = spec["paths"][i][j]
+            if "summary" not in k:
+                print("Error: summary missing from OpenAPI doc in: %s " % (i))
+                errored = True
+    return errored
+
+
+def verify_asyncapi_1(spec):
+    msgs = spec["components"]["messages"]
+    errored = False
+    for name, msg in msgs.items():
+        if "summary" not in msg:
+            print("Error: #components/messages/%s is missing a summary" % name)
```

Comment:
To be consistent, something like:
```suggestion
            print("Error: #components/messages/%s from AsyncAPI is missing a summary" % name)
```